TunnelVisionLabs.ReferenceAssemblyAnnotator/NullableAttributes.cs

```diff
@@ -84,4 +84,61 @@ internal sealed class DoesNotReturnIfAttribute : Attribute
         /// <summary>Gets the condition parameter value.</summary>
         public bool ParameterValue { get; }
     }
+
+    /// <summary>Specifies that the method or property will ensure that the listed field and property members have not-null values.</summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Property, Inherited = false, AllowMultiple = true)]
+    internal class MemberNotNullAttribute : Attribute
```

Comment:
I think these should be declared as sealed. Could you update the URL at the top to https://github.com/dotnet/runtime/blob/v5.0.0-rc.1.20451.14/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs and then compare against it?

Reply:
Nice catch! Looks like I accidentally removed it when copying the definition 😅 

Fixed.